spec/views/users/phone_confirmation/show.html.slim_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe 'users/phone_confirmation/show.html.slim' do
+  before do
+    user = build_stubbed(:user, :signed_up)
+    allow(view).to receive(:current_user).and_return(user)
+  end
+
+  it 'informs the user that a confimation code has been sent' do
+    @unconfirmed_mobile = '12345'
+    render
+
+    expect(rendered).to have_content('A confirmation code has been sent to 12345')
+  end
+end
```

Comment:
Please also add tests for the page title, the h1 heading, and the links to entering the number again and requesting a new code.